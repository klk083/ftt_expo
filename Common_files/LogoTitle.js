import React from 'react';
import {Image} from "react-native";

export default class LogoTitle extends React.Component {
    render() {
        return (
            <Image
                style={{width: 120, height: 120}}
                source={require('../assets/fast_track_taxi_logo_ferdig.png')}
            />
        );
    }
}
