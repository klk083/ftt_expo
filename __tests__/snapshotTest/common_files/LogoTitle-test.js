import React from 'react'
import LogoTitle from '../../../common_files/LogoTitle'

import renderer from 'react-test-renderer'

test('renders correctly', () => {
    const tree = renderer.create(<LogoTitle />).toJSON()
    expect(tree).toMatchSnapshot()
})
