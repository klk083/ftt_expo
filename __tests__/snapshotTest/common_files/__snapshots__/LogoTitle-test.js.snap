// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  resizeMode="cover"
  source={
    Object {
      "testUri": "../../../assets/fast_track_taxi_logo_ferdig.png",
    }
  }
  style={
    Object {
      "height": 107,
      "width": 267,
    }
  }
/>
`;
